docs/qa/CheckLists/SettingsPage.md

```diff
@@ -0,0 +1,40 @@
+|    | 26 Settings                                                                                                                                                |             | Design page 26                                                                                                                          |
+| 1  | Check all UI texts/elements on design page 26                                                                                                              |             |                                                                                                                                         |
+| 2  | Pressing "Sound" radio button enables/disables sound of the app                                                                                            |             | Is it correct? Or does it disable notifications?                                                                                        |
+| 3  | Pressing "Show welcome page" radio button enables/disables appearance of "welcome" page (design page 1)                                                    |             |                                                                                                                                         |
+| 4  | Pressing "Clear Recycle bin every" button opens up a bottom sheet with 4 options, it can be checked on (design page 26.1)                                  |             |                                                                                                                                         |
+| 5  | All 4 options of "Clear Recycle bin every" work as intended                                                                                                |             |                                                                                                                                         |
+| 6  | Pressing "Proceed to Recycle bin" button leads to recycle bin page (design page 28)                                                                        |             |                                                                                                                                         |
+| 7  | Pressing "Help" button leads to help page (design page 27)                                                                                                 |             |                                                                                                                                         |
+| 8  | Pressing "Licenses" button leads to somewhere                                                                                                              |             |                                                                                                                                         |
```

Comment:
to Licenses info

Reply:
Changed to 'Pressing "Licenses" button leads to to Licenses info'